recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -52,14 +50,14 @@ def configure(self):
             self.options["glib"].shared = True
 
     def layout(self):
-        basic_layout(self, src_folder="source")
+        basic_layout(self, src_folder="src")
 
     def validate(self):
-        if self.options.shared and not self.options["glib"].shared:
+        if self.info.options.shared and not self.options["glib"].shared:
```

Comment:
```suggestion
        if self.info.options.shared and not self.info.options["glib"].shared:
```

Reply:
This doesn't work, and results in the error
```
ERROR: gdk-pixbuf/2.42.9: Error in validate() method, line 64
        if self.info.options["glib"].shared and microsoft.is_msvc_static_runtime(self):
        ConanException: option 'shared' doesn't exist
```

I don't entirely follow the explanation of the difference between `self.info.options` and `self.options`, but I assume it is to do with the latter being the options of a dependency

Reply:
In the validate method, you should use the dependencies model to access the options of `glib`. I think something pretty close to the following should work.

```suggestion
        if self.info.options.shared and not self.dependencies.direct_host["glib"].options.shared:
```